website/static/js/filters.js

```diff
@@ -0,0 +1,124 @@
+(function($){
+    $.fn.filters = function (options) {
+        var settings = $.extend({
```

Comment:
There are a lot more options being passed in the implementation of filter like ```settings.inputs```. Your implementation passes these variables but you don't have defaults for this in this plugin nor are there checks to see if they are defined or not. Which means if someone else tries to use this plugin it will break. Can you add defaults and checks to make sure options are properly passed?